system/rest/request/permissions.go

```diff
@@ -66,6 +66,11 @@ type (
 		//
 		// Role ID
 		RoleID uint64 `json:",string"`
+
+		// GenericOnly GET parameter
+		//
+		// Show only generic rules for resources
```

Comment:
Can you explain what "generic" rules are? We did not have this concept before.

Reply:
Generic rules as in more common rule which applies to the resource in general, and not specific to a particular resource.